docs/products/opensearch/reference/index-replication.rst

```diff
@@ -0,0 +1,36 @@
+Automatic adjustment of replication factors
+===========================================
+
+Aiven for OpenSearch automatically adjusts index replication factors
+to ensure data availability and service functionality.
+
+When replication factor is automatically adjusted?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  If ``number_of_replicas`` is too large for current cluster size, it
+   is automatically lowered to maximum possible value (number of nodes
+   on the cluster - 1).
```

Comment:
I would rephrase as something like:
```
The maximum value for ``number_of_replicas`` is the number of nodes in the cluster - 1. 
```
and then carry on with the example from below:
```
For example, for a 3-node cluster, the maximum ``number_of_replicas`` is 2, which means all shards on the index are replicated to all 3 nodes.
```
and finish with
```
If ``number_of_replicas`` is too large, it will automatically be reduced to the maximum value.
```